go-controller/pkg/ovn/loadbalancer/lb_cache.go

```diff
@@ -93,6 +100,50 @@ func (c *LBCache) removeVips(toRemove []DeleteVIPEntry) {
 	}
 }
 
+// RemoveSwitches removes the provided toRemoveSwitches which is a list
+// of switches from all the lb's.Switches in the LBCache.
+func (c *LBCache) RemoveSwitches(toRemoveSwitches []string) {
```

Comment:
Tiny nit: not a fan of the name toRemoveSwitches - would rather just "switches". The method name explains the context - no need to repeat. Up to you.

Reply:
I just followed suit to `removeVIPs`

Reply:
agreed, `switches` is clear in the context.

Reply:
ok I'll update this as well.